transport-msg.tex

```diff
@@ -437,3 +429,249 @@ \subsection{Messages Definition}\label{sec:Virtio Transport Options / Virtio Ove
 \hline
 \end{tabular}
 
+\subsection{Message Bus}\label{sec:Virtio Transport Options / Virtio Over Messages / Message Bus}
+
+The Message Bus acts as a communication channel between two endpoints,
+providing a standardised abstraction to exchange messages.
+
+The Message Bus transmits the messages between the frontend and the backend in
+both directions.
+
+A system can have several Message Busses and it shall respect the following
```

Comment:
"buses" is a much more common spelling than "busses" both in virtio and in general English use.

Reply:
ack